pkg/lib/machines.js

```diff
@@ -805,7 +814,6 @@ cockpit.transport.wait(function() {
         mod.allow_connection_string = $.inArray("connection-string", caps) != -1;
         mod.has_auth_results = $.inArray("auth-method-results", caps) != -1;
         known_hosts_path = "/var/lib/cockpit/known_hosts";
```

Comment:
I guess we can remove this too now, also the debug message. I don't think we've supported ssh to legacy ws for a while now.

Reply:
Ah, good, I was wondering. I'll try to dig out when this has changed, just to be sure and to be able to document it properly.

Reply:
The `"ssh"` capability was removed in 082da24a5a2a27d521e7b6d1beb2bbe094ffdc54, part of #6102, released in version 135, on Mar 16, 2017.

Reply:
Centos 7 has cockpit 195, so I think we are good to remove support for cockpit-ws with "ssh" capability.